portfolio/src/main/webapp/style.css

```diff
@@ -36,6 +36,15 @@ a{
   float:right;
 }
 
+#dynamicDescription{
```

Comment:
Stick to hyphenated css id/classes. 

https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

dynamic-description

Reply:
I will push the revision with that update now